File: index-local.js
var scheduler = require('./index');

async function runGetPoemCheck(){
    // let response = await scheduler.handler({
    //     "body": JSON.stringify({
    //         category: "poem",
    //         wordCount: 45,
    //         genLimit: 3
    //     })
    // });

    // console.log(response);

    // response = await scheduler.handler({
    //     "body": JSON.stringify({
    //         category: "site",
    //         wordCount: 45,
    //         siteName: "ramseysolutions",
    //         genLimit: 3
    //     })
    // });

    // console.log(response);

    // response = await scheduler.handler({
    //     "body": JSON.stringify({
    //         category: "tweets",
    //         wordCount: 45,
    //         genLimit: 3
    //     })
    // });

    // console.log(response);

    response = await scheduler.handler({
        "body": JSON.stringify({
            category: "rapSongs",
            wordCount: 45,
            genLimit: 3
        })
    });

    console.log(response)
    return response;
}

runGetPoemCheck();

File: index-local.py
import json
import datetime
import index

def main():
    runGetDataCheck()

def runGetDataCheck():
    response = index.endpoint({
         "body": json.dumps({
             "category": "songs",
             "wordCount": 45
         })
     });

    print(response)

    # response = index.endpoint({
    #     "body": json.dumps({
    #         "category": "tweets",
    #         "wordCount": 45
    #     })
    # });

    # print(response)

    # response = index.endpoint({
    #     "body": json.dumps({
    #         "category": "rapSongs",
    #         "wordCount": 45
    #     })
    # });

    # print(response)

    return response

if __name__ == "__main__":
    main()
